### ISSUE TITLE
LuaElectricEnergySourcePrototype::effectivity Removal

### ISSUE THREAD

**Issue by TehFreek** [Helfima/helmod#130]
Since the attribute in the title was removed in 0.16.31, Helmod now gives:

```
Script @__helmod__/controller/Controller.lua:273: LuaElectricEnergySourcePrototype doesn't contain key effectivity.
stack traceback:
        __helmod__/model/EntityPrototype.lua:396: in function 'getElectricEffectivity'
        __helmod__/model/Player.lua:790: in function 'getProductionsBeacon'
        __helmod__/edition/AbstractEdition.lua:829: in function 'updateBeaconSelector'
        __helmod__/edition/AbstractEdition.lua:473: in function 'updateBeacon'
        __helmod__/edition/AbstractEdition.lua:361: in function 'onUpdate'
        __helmod__/dialog/Dialog.lua:223: in function 'update'
        __helmod__/dialog/Dialog.lua:123: in function 'open'
        __helmod__/dialog/Dialog.lua:155: in function 'sendEvent'
        __helmod__/controller/Controller.lua:294: in function 'sendEvent'
        __helmod__/controller/Controller.lua:266: in function <__helmod__/controller/Controller.lua:205>
```

**Comment by mmysinger** [comment]
Changing the getElectricEffectivity function to just return 0 fix this for me.

**Comment by Helfima** [comment]
fixed in v0.7.8
